src/services/Migrations.ts

```diff
@@ -40,6 +41,26 @@ class Migrations {
       console.log(`collection ${CachedValidator} removed.`);
     });
   };
+
+  private static migrateTo_7_28_0 = async () => {
+    await Migrations.wrapMigration('7.28.0', async () => {
```

Comment:
Matheus already used this migration ID, so to not have conflicts  please use `7.27.1`